pages/index.tsx

```diff
@@ -1,5 +1,8 @@
+import { useRouter } from 'next/router'
+
 const DIDAdminPanel = () => {
-  return <h1>DID Admin Panel</h1>
+  const { push } = useRouter()
+  push('/eventos')
```

Comment:
I think this needs a return. One option is to implement redirect instead

https://nextjs.org/docs/api-reference/next.config.js/redirects

Reply:
I implemented [get-static-props](https://nextjs.org/docs/basic-features/data-fetching/get-static-props)